content/en/service_management/workflows/private_actions/use_private_actions.md

```diff
@@ -76,6 +76,20 @@ From the **Private Action Runner** page in [Workflow Automation][6] or [App Buil
    You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
 
 [101]: https://docs.docker.com/compose/compose-application-model/
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+1. Click **Kubernetes**.
+1. Confirm that you have installed `kubectl` on your machine by running `kubectl version` and verifying that there is output, then check the box on the **Private Action Runner** page.
+1. Confirm that you have installed `helm` on your machine by running `helm version` and verifying that there is output, then check the box on the **Private Action Runner** page.
+1. Confirm that you have sufficient permissions to create Kubernetes resources in your cluster by running `kubectl get pods` and verifying that the output is not an error, then check the box on the **Private Action Runner** page.<br>Further instructions appear in the app.
```

Comment:
Technically running `kubectl get pods` is not sufficient to guarantee that you have all the required permissions. The exact permissions are a bit different but I don't know how much details we want to provide ? They might also change based on future updates so I'm not sure being super specific would be helpful.

Reply:
Okay, I can make this less specific :)